bin/junit/TEST-za.ac.sun.cs.green.EntireSuite.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Don't push your bin directory. You only want to check in source code into your remote repo.